pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+markers =
+  controller_data
+  handler_data
```

Comment:
These markers are used to provide 'parametrization' to some fixtures.

The technique is [explained here](https://docs.pytest.org/en/stable/fixture.html#using-markers-to-pass-data-to-fixtures)